internal/armada/scheduling/pod_matching_context.go

```diff
@@ -2,29 +2,28 @@ package scheduling
 
 import (
```

Comment:
do we still need this? I had a feeling this was used by the old scheduler but do w e use it in the new one?

Reply:
It's still used by the submit-time checks I believe.

Reply:
In any case, let's just fix compiler errors resulting from me moving things around now and remove the code subsequent PRs.